fix: US132834, styling changes that came up in testing

martinwang-d2l opened the pull request.

dustxd commented on components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-submission-views-access-restrictions-editor.js:
```diff
@@ -126,9 +132,9 @@ class ActivityQuizSubmissionViewsAccessRestrictionsEditor
 			${attemptRestrictions ? html`
 			<d2l-input-checkbox-spacer>
-				<label class="d2l-label-text">
+				<div class="d2l-label-text">
```
why are we changing this back to a `<div>`?

martinwang-d2l replied:
The label makes the header appear alongside the dropdown instead of above it

dustxd replied:
add a styling class and change the display to block should fix it

dustxd replied:
It's good to give the elements a semantic meaning whenever, compared to using `<div>` or `<span>` 🙂

martinwang-d2l replied:
Ah yeah that fixes it thanks

martinwang-d2l commented:
New spacing: 
![image](https://user-images.githubusercontent.com/88114520/145116174-9a3587fc-2db7-478b-8c67-9e1caa58eb9f.png)

Items fixed: 
- Fixed typo in "A specific attempt" lang term 
- Moved "Attempt" label to correct position
- Fixed spacing between percentage boxes 
- Increased "minute(s)" font size
